insights/specs/sos_archive.py

```diff
@@ -88,6 +88,9 @@ class SosSpecs(Specs):
     pcs_config = simple_file("sos_commands/pacemaker/pcs_config")
     pcs_quorum_status = simple_file("sos_commands/pacemaker/pcs_quorum_status")
     pcs_status = simple_file("sos_commands/pacemaker/pcs_status")
+    podman_list_containers = first_file(["sos_commands/podman/podman_ps_-a", "sos_commands/podman/podman_ps"])
```

Comment:
Would you please keep them in the same order with them in the `__init__.py` file?

Reply:
done